### PR TITLE
Implement parser for AddAssignmentCommand

### LINKED ISSUE #28: Create module/class-wide assignment
### User Story

As a user, I can create a module/class-wide assignment, with information such as due date, maximum marks/grade, weightage so that I am able to keep track of this assignment for each student.

### PR DESCRIPTION
**Pull request by marcusc55**
`AddAssignmentCommandParser` class added to handle parsing name, weightage and maximum score from the user's input.

### CODE REVIEW

File: src/main/java/seedu/edrecord/logic/parser/AddAssignmentCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import seedu.edrecord.model.assignment.Assignment;
+import seedu.edrecord.model.assignment.MaxScore;
+import seedu.edrecord.model.assignment.Weightage;
+import seedu.edrecord.model.name.Name;
+
+public class AddAssignmentCommandParser implements Parser<AddAssignmentCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the {@code MakeModuleCommand}
+     * and returns a {@code MakeModuleCommand} object for execution.
```
Comment (uyencfi) [inline src/main/java/seedu/edrecord/logic/parser/AddAssignmentCommandParser.java:21]: Should it be `AddAssignmentCommand` in the header comment?
```suggestion
     * Parses the given {@code String} of arguments in the context of the {@code AddAssignmentCommand}
     * and returns a {@code AddAssignmentCommand} object for execution.
```

File: src/main/java/seedu/edrecord/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -156,4 +158,34 @@ public static Module parseMakeModule(String moduleCode) throws ParseException {
...
+        }
+        return new Weightage(trimmedWeightage);
+    }
+
+    /**
+     * Parses a {@code String weightage} into a {@code Weightage}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code weightage} is invalid.
+     */
```
Comment (uyencfi) [inline src/main/java/seedu/edrecord/logic/parser/ParserUtil.java:182]:
```suggestion
    /**
     * Parses a {@code String maxScore} into a {@code MaxScore}.
     * Leading and trailing whitespaces will be trimmed.
     *
     * @throws ParseException if the given {@code maxScore} is invalid.
     */
```

**Review by uyencfi — APPROVED** [review]
Other than minor changes to the header comments, LGTM!

**Review by chaiwanlin — APPROVED** [review]
lgtm

**Pull request merged**